flexget/plugins/clients/qbittorrent.py

```diff
@@ -225,6 +228,19 @@ def add_entries(self, task, config):
             if label:
                 form_data['label'] = label  # qBittorrent v3.3.3-
                 form_data['category'] = label  # qBittorrent v3.3.4+
+            try:
+                tags = entry.render(
+                    ",".join(
+                        [",".join(config.get('tags', '')), ",".join(entry.get('tags', ''))]
+                    ).strip(",")
+                )
+                if tags:
+                    form_data['tags'] = tags
+            except RenderError as e:
+                logger.error('Error rendering tags for {}: {}', entry['title'], e)
+                form_data['tags'] = ",".join(
+                    [",".join(config.get('tags', '')), ",".join(entry.get('tags', ''))]
+                ).strip(",")
```

Comment:
With most other settings, the field from the entry will override what was provided in the config, rather than augment it. I think we should keep this the same for consistency (unless you have a compelling reason otherwise.) Also, it's probably nicer to put an if clause around this whole thing, so that we don't try to do any rendering in the case where no tags were specified.

i.e. something like this to start off:
```python
tags = entry.get('tags', config.get('tags', []))
if tags:
    ...
```

Reply:
The reason I didn't want to override a config tag with an entry tag is simply because there can be multiple tags (as opposed to label och savepath that only have one). This gives the user the possibility to tag all torrents with something very general for every torrent from the config (eg "Flexget") but then also add specific tags from the entry.
Does it make sense?

Reply:
Unrelated note: Would it be cleaner to merge both tag lists before converting to string?
```python
form_data['tags'] = ",".join(config.get('tags', [])+entry.get('tags', []))
```

Reply:
> The reason I didn't want to override a config tag with an entry tag is simply because there can be multiple tags (as opposed to label och savepath that only have one). This gives the user the possibility to tag all torrents with something very general for every torrent from the config (eg "Flexget") but then also add specific tags from the entry.
Does it make sense?

Yeah, I'm okay with that.

I think we should still make the master list of tags, check if it's not empty, then start doing all the rendering and formatting.

Reply:
Similar to @BrutuZ  idea. Something like:
```python
tags = entry.get('tags', []) + config.get('tags', [])
if tags:
    ...
```

Reply:
Ok thanks for input. Im then going to change to this:

```
            tags = entry.get('tags', []) + config.get('tags', [])
            if tags:
                try:
                    form_data['tags'] = entry.render(",".join(tags))
```
I Agree looks better so thanks the suggestion @BrutuZ